submissions/yevheniiii/html_css_popup/index.html

```diff
@@ -0,0 +1,147 @@
+                        </a>
+                    </li>
+
+                    <li class="inside-item">
+                        <a href="#">
+                            <img src="images/bell.png" alt="bell" />
+                            <section class="items-description">Bell</section>
+                        </a>
+                    </li>
+
+                    <li class="inside-item space"></li>
+                </ul>
+
+                <input type="checkbox" id="buttonMore" class="MoreApps" />
+                <label for="buttonMore" class="MorePopUp">More</label>
+
+                <hr class="line" />
+
+                <ul class="more-items">
+                    <li class="inside-item hidden">
+                        <a href="#">
+                            <img src="images/plus.png" alt="plus" />
+                            <section class="items-description">Plus</section>
+                        </a>
+                    </li>
+
+                    <li class="inside-item hidden">
+                        <a href="#">
+                            <img src="images/plus.png" alt="plus" />
+                            <section class="items-description">Plus</section>
+                        </a>
+                    </li>
+
+                    <li class="inside-item hidden">
+                        <a href="#">
+                            <img src="images/plus.png" alt="plus" />
+                            <section class="items-description">Plus</section>
+                        </a>
+                    </li>
+
+                    <li class="inside-item hidden">
+                        <a href="#">
+                            <img src="images/plus.png" alt="plus" />
+                            <section class="items-description">Plus</section>
+                        </a>
+                    </li>
+
+                    <li class="inside-item hidden">
+                        <a href="#">
+                            <img src="images/plus.png" alt="plus" />
+                            <section class="items-description">Plus</section>
+                        </a>
+                    </li>
+
+                    <li class="inside-item hidden">
+                        <a href="#">
+                            <img src="images/plus.png" alt="plus" />
+                            <section class="items-description">Plus</section>
+                        </a>
+                    </li>
+                </ul>
+            </section>
+
+            <section class="nav-item">
+                <a><img src="images/bell.png" alt="user-notifications" /></a>
+            </section>
+
+            <section class="nav-item">
+                <a><img class="user-avatar" src="images/img_avatar.png" alt="user-avatar" /></a>
+            </section>
+
+            <section class="popup-triangle">
+                <section class="empty"></section>
+            </section>
+        </section>
+    </nav>
+</body>
+
+</html>
```

Comment:
Add an empty line at the end of each file.

Reply:
I did this. 

Can you explain why we should add an empty line?
@A-Ostrovnyy

Reply:
[Read more](https://stackoverflow.com/questions/729692/why-should-text-files-end-with-a-newline?noredirect=1&lq=1)

Reply:
Shame on me that I didn't find it on Google.
Thanks!